index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Project</title>
+</head>
+<body>
+
```

Comment:
Please do your best to replicate the given design, the margins are not same 

![temp1 student](https://user-images.githubusercontent.com/59707859/227044786-57ca213e-bd5b-4659-a138-22b088bcb634.PNG)
![temp1](https://user-images.githubusercontent.com/59707859/227044790-6453f80d-eb3f-49c4-ae38-007b4a5038a1.PNG)

Reply:
@souad988  Resolved The Margin Issue; You can recheck.

Reply:
![margin-issue-resolve](https://user-images.githubusercontent.com/124990680/227066653-5757f6db-3810-42ca-8a33-a081e3e92baa.PNG)